.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+**I'm submitting a ...**  (check one with "x")
+- [ ] bug report
+- [ ] feature request
+- [ ] support request
+
+**Current behavior:**
+<!-- Describe how the bug manifests. -->
+
+**Expected behavior:**
+<!-- Describe what the behavior would be without the bug. -->
+
+**Steps to reproduce:**
+<!-- If you are able to illustrate the bug or feature request with an example, please provide steps to reproduce -->
+
+**Related code:**
+
+```
```

Comment:
Code block should not be wrapped in a comment?

Reply:
Why in a comment?